content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,126 @@
+> 
+> * short discussion about the two "paths" for installing kubeflow "XXXXX" and "YYYYY"
+
+<a id="packaged-distributions"></a>
+## Packaged distributions
+
+> TEXT TO WRITE:
+> 
+> * distributions are separate from the kubeflow community
+>    * none are currently endorsed or certified
+>    * please reach out to the maintainers to get support
+> * the method for installing can vary from distribution to distribution, please consult the docs for your chosen distribution    
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Name</th>
+        <th>Maintainer</th></th>
+        <th>Platform</th>
+        <th>Link - Docs</th>
+        <th>Link - External Website</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Kubeflow on AWS</td>
+        <td>AWS</td>
+        <td>Amazon Elastic Kubernetes Service (EKS)</td>
+        <td><a href="/docs/distributions/aws/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Azure</td>
+        <td>Azure</td>
+        <td>Azure Kubernetes Service (AKS)</td>
+        <td><a href="/docs/distributions/azure/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on GCP</td>
+        <td>Google Cloud</td>
+        <td>Google Kubernetes Engine (GKE)</td>
+        <td><a href="/docs/distributions/gke/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on IBM Cloud</td>
+        <td>IBM Cloud</td>
+        <td>IBM Cloud Kubernetes Service (IKS) </td>
+        <td><a href="/docs/distributions/ibm/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Openshift</td>
+        <td>IBM Cloud</td>
+        <td>Openshift</td>
+        <td><a href="/docs/distributions/openshift/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Argoflow</td>
+        <td>Argoflow Community</td>
+        <td>Conformant Kubernetes</td>
+        <td>N/A</td>
+        <td><a href="https://github.com/argoflow/argoflow">External Website</a></td>
+      </tr>
+      <tr>
+        <td>Charmed Kubeflow</td>
+        <td>Canonical</td>
+        <td>Conformant Kubernetes</td>
+        <td><a href="/docs/distributions/charmed/">Docs</a></td>
+        <td><a href="https://charmed-kubeflow.io/docs">External Website</a></td>
+      </tr>
+      <tr>
+        <td>MicroK8s Kubeflow add-on</td>
+        <td>Canonical</td>
+        <td>MicroK8S</td>
+        <td><a href="/docs/distributions/microk8s/">Docs</a></td>
+        <td><a href="https://microk8s.io/docs/addon-kubeflow">External Website</a></td>
+      </tr>
+      <tr>
+        <td>MiniKF</td>
+        <td>Arrikto</td>
+        <td>Cloud Marketplaces, Vagrant</td>
+        <td><a href="/docs/distributions/minikf/">Docs</a></td>
+        <td></td>
+      </tr>
+    </tbody>
```

Comment:
Curious if the community is interested in adding an `on-prem kubeflow installation` guide?

Reply:
Lets just leave this a distributions for now, but if there is an on-prem distribution, we could link it here.

Reply:
I think there are multiple on-prem options, so we could at most tag them. Providing 1 guide will likely favour one option.

Reply:
I agree that we can discuss later on. We don't need to take action on linking them for this release.